README.md

```diff
@@ -651,7 +651,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Daniel Lawrence](https://github.com/DanLawrence91)
   - [Daniel Mutwiri](https://github.com/gitonga123)
   - [Daniel Perkins](https://github.com/PerkinsDan)
-  - [Danyal Ahmad](https://github.com/danyalahmad04)
+  - [Danysal Ahmad](https://github.com/danyalahmad04)
```

Comment:
I don't think you should update others' profiles. 🤔

Reply:
The PR creator's name is Daniel, while the changed name is Danysal. 🤔

Reply:
But both the links are same :)

Reply:
The links seem to be different here (`danyalahmad04` and `DanielToucedo`).